src/EnvConfig.php

```diff
@@ -38,7 +41,6 @@ class EnvConfig implements SiteConfig
     private const HOSTING_LOCATIONS_CLASS_MAP = [
         self::HOSTING_WPE => WpEngineLocations::class,
         self::HOSTING_VIP => VipLocations::class,
-        self::HOSTING_SPACES => GenericLocations::class,
```

Comment:
Why removing it? I mean, even if _we_ don't use it maybe someone else will? This is a public repo...

Reply:
Spaces hosting does not exist anymore, they closed the service. :)